packages/api/src/server.js

```diff
@@ -298,6 +301,9 @@ app
   .use('/sale', saleRoutes)
   .use('/revenue_type', revenueTypeRoute)
   .use('/task_type', taskTypeRoutes)
+  .use('/soil_amendment_purpose', soilAmendmentPurposeRoute)
```

Comment:
I'm not super fussed, but we used plural endpoint names with animals (e.g. `animal_uses` etc.) -- do you remember if that was supposed to be a new pattern for all new routes?

Actually on that topic (and it's 100% my doing, sorry) probably `soil_amendment_task_products` (table) should have been named in the singular 😬

Reply:
Good question, I can update. Yeah the table naming had me confused too -- we need a different naming convention it gets quite wordy!

Reply:
Got some answers from Shubleen on https://lite-farm.atlassian.net/browse/LF-4246 so will update the table name there -- added "extras" to dev tasks.